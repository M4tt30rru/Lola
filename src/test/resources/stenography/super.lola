##Import "std.lola"
##Find
 ##Identifier(n)(##ArgumentDeclarationList(l)) -> super;
 ##replace ##(n) (##(l)) {super(##(",".join(a.n.name for a in l.args)));}

T(int x, Double y) -> super;
