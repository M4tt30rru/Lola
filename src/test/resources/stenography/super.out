T(int x, Double y) {
 super(x, y);
}
